docs/source/quickstart.rst

```diff
@@ -0,0 +1,108 @@
+Quickstart
+==========
+
+This guide shows how to create, compile, and run a simple pipeline with Kubeflow Pipelines (KFP).
+
+Prerequisites
+-------------
+- Python 3.6+
```

Comment:
```suggestion
- Python 3.9+
```

though note, after yesterday's community call this will very soon be Python 3.10+, I believe